### DISCUSSION TITLE
Missing v1.16.0-RC1 artifacts

### DISCUSSION THREAD

**Discussion by natiginfo** [detekt/detekt#3425] [Q&A] [answered]
In previous releases of detekt, there used to be `detekt` besides source code in assets. However, in v1.16.0-RC1, only source code is available in assets. Is it a mistake?

**Comment by cortinico** [comment, marked as answer]
Thanks for the report. That's actually true. I've updated the `v1.16.0-RC1` release adding the artifacts manually.
Ideally this https://github.com/detekt/detekt/discussions/3427 will fix the issue for the upcoming releases.

**Reply by BraisGabin** [reply]
And #3254 removed one of the artifacts too. We need to find a way to fix that.
